package.json

```diff
@@ -24,8 +24,10 @@
     "Dao Lam <daolam112@gmail.com>",
     "Darren Matsumoto <aeneascorrupt@gmail.com>",
     "Jeremiah Wuenschel <jeremiah.wuenschel@gmail.com>",
+    "Jerry Zhang <thejerryzhang@gmail.com>",
```

Comment:
You may want to consider sending a PR to [screwdriver-cd/generator-screwdriver](https://github.com/screwdriver-cd/generator-screwdriver) so this will be automagic